ts/src/profilers/profile-serializer.ts

```diff
@@ -209,7 +235,8 @@ function createAllocationValueType(table: StringTable):
  * @param intervalMicros - average time (microseconds) between samples.
  */
 export function serializeTimeProfile(
-    prof: TimeProfile, intervalMicros: number): perftools.profiles.IProfile {
+    prof: TimeProfile, intervalMicros: number,
```

Comment:
nit: add `hasLines` to the jsdoc comment above as well.

Reply:
Done.